app/javascript/redux/getText.js

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+const getText = createAsyncThunk("greeting", async () => {
+  try {
+    const text = await fetch("http://127.0.0.1:3000/api/greetings");
+    const data = await text.json();
+    if (data.text) {
+      return {
+        success: true,
+        data:data.text,
+      };
+    }
+  } catch (err) {
+    return { success: false, err: err.response.data.text };
+  }
+});
```

Comment:
- [ ] When I run your application, I get this error in the console:
 
![Screenshot 2023-07-10 173433](https://github.com/mbdesigns1989/Hello_Rails_React/assets/77106962/348ca9b7-743f-48a6-bec5-609713c92c42)

I would advise you to fetch the Greetings data using this URL `"http://localhost:3000/api/greetings"`. 💻